rust/src/gildedrose.rs

```diff
@@ -88,12 +109,208 @@ impl GildedRose {
 mod tests {
     use super::{GildedRose, Item};
 
+    pub struct TestData {
```

Comment:
This is a nice way to handle the tests. I tried to create a new struct type like this to handle the requests but ran into borrow issues.